internal/pwru/types.go

```diff
@@ -68,6 +70,8 @@ func (f *Flags) SetFlags() {
 
 	flag.StringVar(&f.ReadyFile, "ready-file", "", "create file after all BPF progs are attached")
 	flag.Lookup("ready-file").Hidden = true
+
+	flag.StringVar(&f.Backend, "backend", "", "Tracing backend('kprobe', 'kprobe-multi')")
```

Comment:
Maybe worth mentioning the the usage that this is auto-detected in case backend is empty and kmods are empty. At least that's how I read the following:

https://github.com/cilium/pwru/pull/99/files#diff-2873f79a86c0d8b3335cd7731b0ecf7dd4301eb19a82ef7a1cba7589b5252261R92-R93